src/components/shared/components/Layout/components/NavigationLayout.tsx

```
@@ -1,170 +1,138 @@
+                          }}
+                        />
+
+                        {navPages.find((page) => page.route === item.url) && (
+                          <List className="w-full">
+                            {navPages
+                              .filter((page) => page.route === item.url && "children" in page)
+                              .map((page) =>
+                                page.children
+                                  .filter((page) => page.route !== item.url)
+                                  .map((page) => (
+                                    <div className="px-3 pl-12 sm:pr-6">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)